### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by pnowosie**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

I've recently run setup on clean Ubuntu machine just to find our instruction are broken.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -9,33 +9,22 @@ Only **Linux** and **OSX** platforms are supported now. These instructions have
...
+sudo apt-get update
 sudo apt-get install esl-erlang=1:22.3.1-1 elixir=1.10.2-1
-sudo apt-get install -y erlang-os-mon
-```
-
-## Install Geth
-Install Geth version 1.9.12 from the URL below.
-```
-https://geth.ethereum.org/downloads/
+sudo apt-get install -y erlang-os-mon erlang-parsetools
```
Comment (InoMurko) [inline docs/install.md:27]:
```suggestion
sudo apt-get install -y erlang-os-mon erlang-parsetools erlang-tools
```

Comment (InoMurko) [reply docs/install.md:27]: cover - A coverage analysis tool for Erlang.
 cprof - A profiling tool that shows how many times each function
   is called. Uses a kind of local call trace breakpoints containing
   counters to achieve very low runtime performance degradation.
 eprof - A time profiling tool; measures how time is used in Erlang
   programs. Predecessor of fprof (see below).
 fprof - Another Erlang profiler; measures how time is used in Erlang
   programs. Uses trace to file to minimize runtime performance
   impact, and displays time for calling and called functions.
 instrument - Utility functions for obtaining and analyzing resource
   usage in an instrumented Erlang runtime system.
 make - A make utility for Erlang similar to UNIX make.
 tags - A tool for generating Emacs TAGS files from Erlang source
   files.
 xref - A cross reference tool. Can be used to check dependencies
   between functions, modules, applications and releases.

**Comment by InoMurko** [comment]
> Please update `bin/setup` to install geth 1.9.12

I think it does already:
```
install_linux_geth(){
  echo "Installing Geth 1.9.12"
  mkdir gethDownload \
  && wget -qO- "https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.9.12-b6f1c8dc.tar.gz" | tar xvz -C gethDownload --strip-components 1 \
  && cd gethDownload \
  && mkdir -p /usr/local/bin \
  && sudo mv geth /usr/local/bin/ \
  && cd .. \
  && rm -r gethDownload
}
```

**Comment by pgebal** [comment]
> > Please update `bin/setup` to install geth 1.9.12
> 
> I think it does already:
> 
> ```
> install_linux_geth(){
>   echo "Installing Geth 1.9.12"
>   mkdir gethDownload \
>   && wget -qO- "https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.9.12-b6f1c8dc.tar.gz" | tar xvz -C gethDownload --strip-components 1 \
>   && cd gethDownload \
>   && mkdir -p /usr/local/bin \
>   && sudo mv geth /usr/local/bin/ \
>   && cd .. \
>   && rm -r gethDownload
> }
> ```

True, I have not pulled recent master.

**Comment by pnowosie** [comment]
> Please update `bin/setup` to install geth 1.9.12

I think you did it recently

**Pull request merged**